Add Chainguard and Docker CE File (#294)
odidev/ecosystem-dashboard-for-arm: content/opensource_packages/docker-ce.md

---
name: Docker CE
category: Containers and Orchestration
description: Docker CE (Community Edition) is an open-source platform for developing, shipping, and running applications in containers.
download_url: https://github.com/docker-archive/docker-ce/tags
works_on_arm: true
supported_minimum_version: 
    version_number: 17.09.0
    release_date: 2017/09/27

optional_info:
    homepage_url: https://www.docker.com/
    support_caveats:
    alternative_options: 
    getting_started_resources:
        arm_content: https://learn.arm.com/install-guides/docker/
        partner_content: https://www.alibabacloud.com/blog/arm-container-applications-accelerating-development-and-testing_595802
        official_docs: https://www.docker.com/blog/getting-started-with-docker-for-arm-on-linux/
    arm_recommended_minimum_version:
        version_number:
        release_date:

optional_hidden_info:
    release_notes__supported_minimum: https://docs.docker.com/engine/release-notes/17.09/
    release_notes__recommended_minimum:

>     other_info: From the list available at [Docker's official download page](https://download.docker.com/linux/static/stable/aarch64/) for ARM64, it is clear that Docker CE 17.09.0 was among the first versions to offer initial support for ARM64 architecture.

Author: ```suggestion
    other_info: From the list available at [Docker's official download page](https://download.docker.com/linux/static/stable/aarch64/) for ARM64, it is clear that Docker CE 17.09.0 rolled out initial support for ARM64 architecture.
```
